src/Symfony/Bridge/Doctrine/IdGenerator/UuidGenerator.php

```diff
@@ -21,7 +21,7 @@
 
 final class UuidGenerator extends AbstractIdGenerator
 {
-    private UuidFactory $protoFactory;
+    private $protoFactory;
     private UuidFactory|NameBasedUuidFactory|RandomBasedUuidFactory|TimeBasedUuidFactory $factory;
```

Comment:
Union types are not concerned? :thinking:

Reply:
yes they are

Reply:
I guess the removal was done with a search and replace with a regex that was not allowing `?` or `|` when matching the type...